packages/demand-bloc-payment/src/providers/BraintreeProvider.ts

```diff
@@ -192,6 +196,7 @@ export class BraintreeProvider implements Provider {
 
     const { iframeContainerId, loadingId, processingId } = threeDSecureFields
 
+    // @ts-ignore: bin property is no longer optional in ThreeDSecureVerifyPayload type
```

Comment:
What exactly does it mean? If it is not optional we should pass it and remove this comment.
We can also stick to old version of Braintree package for now to avoid payment issues

Reply:
after bumping lib `bin` property is needed in types but works as expected without it (as before the bump) and we decided to leave bumped version and ignore that for now for future development

btw are we using braintree on htbv2?

Reply:
In theory yes - it depends on payment provider configuration.
It is however used for bookers v1 which consumes demand packages.

Maybe let's keep old versions of braintree for now and bump it separately